docs/pages/enroll-resources/application-access/cloud-apis/aws-console-roles-anywhere.mdx

```diff
@@ -0,0 +1,363 @@
+---
+title: AWS Console and CLI Access with Roles Anywhere
+description: How to use AWS IAM Roles Anywhere with Teleport for AWS Console and CLI access
+labels:
```

Comment:
```suggestion
tags:
```
We changed the `labels` frontmatter field to `tags` to accommodate the built-in Docusaurus tagging feature, but hadn't backported it to v18.